docs/fields.md

```diff
@@ -1295,6 +1317,392 @@ CronWorkflowStatus is the status of a CronWorkflow
 |`phase`|`string`|v3.6 and after: Phase is an enum of Active or Stopped. It changes to Stopped when stopStrategy.condition is true|
 |`succeeded`|`integer`|v3.6 and after: Succeeded counts how many times child workflows succeeded|
 
+## WorkflowEventBindingSpec
+
+_No description available_
+
+<details markdown>
+<summary>Examples with this field (click to open)</summary>
+
+- [`archive-location.yaml`](https://github.com/argoproj/argo-workflows/blob/main/examples/archive-location.yaml)
```

Comment:
FYI: These examples shouldn't be here, since they don't reference `WorkflowEventBinding` at all. This is also present in every other CRD with `spec` field because the code that generates this page assumes `spec` means the same for all CRDs: https://github.com/argoproj/argo-workflows/blob/948127c7938b32a9a56ff74e31c2367d694d876d/hack/docs/fields.go#L302-L306

Reply:
The other 4 CRDs are the same though (more or less), so it sounds like there needs to be a change to the script to handle this CRD properly

Reply:
Okay, I pushed a hack in cf20a755b54aa41b643fdfbf9ec357f2338aace8 to exclude examples containing `"spec"` from anything except `WorkflowSpec` and `CronWorkflowSpec`